allennlp/tests/training/metrics/span_based_f1_measure_test.py

```diff
@@ -235,24 +244,24 @@ def test_span_f1_matches_perl_script_for_continued_arguments(self):
         assert metric._true_positives["V"] == 1
         assert metric._true_positives["ARGM-ADJ"] == 1
 
-        numpy.testing.assert_almost_equal(metric_dict["recall-ARG1"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["precision-ARG1"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["f1-measure-ARG1"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["recall-V"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["precision-V"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["f1-measure-V"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["recall-ARGM-ADJ"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["precision-ARGM-ADJ"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["f1-measure-ARGM-ADJ"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["recall-overall"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["precision-overall"], 1.0)
-        numpy.testing.assert_almost_equal(metric_dict["f1-measure-overall"], 1.0)
+        assert_allclose(metric_dict["recall-ARG1"], 1.0)
+        assert_allclose(metric_dict["precision-ARG1"], 1.0)
+        assert_allclose(metric_dict["f1-measure-ARG1"], 1.0)
+        assert_allclose(metric_dict["recall-V"], 1.0)
+        assert_allclose(metric_dict["precision-V"], 1.0)
+        assert_allclose(metric_dict["f1-measure-V"], 1.0)
+        assert_allclose(metric_dict["recall-ARGM-ADJ"], 1.0)
+        assert_allclose(metric_dict["precision-ARGM-ADJ"], 1.0)
+        assert_allclose(metric_dict["f1-measure-ARGM-ADJ"], 1.0)
+        assert_allclose(metric_dict["recall-overall"], 1.0)
+        assert_allclose(metric_dict["precision-overall"], 1.0)
+        assert_allclose(metric_dict["f1-measure-overall"], 1.0)
 
         # Check that the number of true positive ARG1 labels is the same as the perl script's output:
         gold_file_path = os.path.join(self.TEST_DIR, "gold_conll_eval.txt")
         prediction_file_path = os.path.join(self.TEST_DIR, "prediction_conll_eval.txt")
-        with open(gold_file_path, "a+") as gold_file, open(
-            prediction_file_path, "a+"
+        with open(gold_file_path, "w") as gold_file, open(
+            prediction_file_path, "w"
```

Comment:
I changed this because the test failed (function called twice), and also the append wasn't necessary.